lib/src/replaced_element.dart

```diff
@@ -90,6 +90,11 @@ class ImageContentElement extends ReplacedElement {
   }
 }
 
+/// Create a webview controller and old src independent of [IframeContentElement]
```

Comment:
🤔  What if you have two iframes? Does this work (independently)?

Reply:
Hah that's something very important that I overlooked. I'm going to mark this as a draft and come back to it.

Reply:
@erickok turns out you only need to assign a `key` to make it work right, as @The-Redhat commented on #337. Now it should work independently!

Reply:
Wait but now the whole Html widget is controlled by the single key... What is the impact of this?

Reply:
I only assigned the key to the WebView. Each WebView will get a different key when a new `IframeContentElement` class is created.